src/models.jl

```diff
@@ -637,16 +639,16 @@ Train the machine using `fit!(mach, rows=...)`.
 
 $KNNFIELDS
 
-* `output_type::Type{<:MultiUnivariateFinite}=DictTable` : One of 
-    (`ColumnTable`, `DictTable`). The type of table type to use for predictions.
+* `output_type::Type{<:MultiUnivariateFinite}=DictColumnTable` : One of 
```

Comment:
In docs I recommend you qualify `DictColumnTable` as `Tables.DictColumnTable` or otherwise explain that this type is owned by 3rd party package.